cmd/telepilot/main.go

```diff
@@ -2,26 +2,133 @@ package main
 
 import (
 	"context"
+	"fmt"
 	"log" // TODO: Consider using slog.
+	"os"
+	"path"
 
-	"google.golang.org/grpc"
-	"google.golang.org/grpc/credentials/insecure"
+	"github.com/google/uuid"
+	"github.com/urfave/cli/v3"
 
-	pb "go.creack.net/telepilot/api/v1"
+	"go.creack.net/telepilot/pkg/apiclient"
+	"go.creack.net/telepilot/pkg/tlsconfig"
 )
 
+//nolint:funlen // Acceptable for CLI definition.
 func main() {
-	conn, err := grpc.NewClient("localhost:9090", grpc.WithTransportCredentials(insecure.NewCredentials()))
-	if err != nil {
-		log.Fatalf("grpc new client: %s.", err)
-	}
-	defer func() { _ = conn.Close() }() // Best effort.
+	var client *apiclient.Client
+	var jobID uuid.UUID
 
-	client := pb.NewTelePilotServiceClient(conn)
+	jobIDArg := &cli.StringArg{
+		Name:      "<job_id>",
+		UsageText: "<job_id>",
+		Min:       1,
+		Max:       1,
+	}
+	parseJobID := func(_ context.Context, cmd *cli.Command) error {
+		id, err := uuid.Parse(cmd.Args().First())
+		if err != nil {
+			return fmt.Errorf("invalid <job_id>, not a uuid: %w", err)
```

Comment:
Should this be a hard error? The job ID in the proto is just a string, it would not be a breaking change to use a different job ID format from the point of view of the server, so the client hard erroring out violates forwards compatibility here.

Reply:
Done.